src/components/Navbar/Mobile/ResourcesDropdownMobile.tsx

```diff
@@ -0,0 +1,99 @@
+import { t, Trans } from '@lingui/macro'
+import { Collapse, Space } from 'antd'
+import CollapsePanel from 'antd/lib/collapse/CollapsePanel'
+import ExternalLink from 'components/shared/ExternalLink'
+import { useState } from 'react'
+
+import { DownOutlined, UpOutlined } from '@ant-design/icons'
+
+import { navMenuItemStyles } from '../navStyles'
+
+function ResourcesItem({
+  text,
+  route,
+  onClick,
+}: {
+  text: string
+  route?: string
+  onClick?: VoidFunction
+}) {
+  return (
+    <ExternalLink
+      className="nav-dropdown-item"
+      href={route}
+      onClick={onClick}
+      style={{ ...navMenuItemStyles, fontWeight: 400, marginTop: 0 }}
+    >
+      {text}
+    </ExternalLink>
+  )
+}
+
+export default function ResourcesDropdownMobile() {
+  const [activeKey, setActiveKey] = useState<0 | undefined>()
+  const iconSize = 12
+
+  // Close dropdown when clicking anywhere in the window
+  window.addEventListener('click', () => setActiveKey(undefined), false)
```

Comment:
**issue** these event listeners will get added every time the dropdown is opened. This might lead to unexpected behavior.

One solution would be to have a hook - something like `useWindowClick`, that only adds the event listener _once_. Then, we can use that hook throughout these components where necessary.


https://user-images.githubusercontent.com/94939382/158013814-f506c483-5c96-43ed-be86-8cbaa7577581.mp4

Reply:
Yep good pickup. I changed it to how @torvusbug suggested we handle a similar previous instance of this. Wdyt? 

![Screen Shot 2022-03-16 at 2 02 43 pm](https://user-images.githubusercontent.com/96150256/158514958-95e7fcf6-6f2c-43a9-abcb-8b76bda26bcc.png)